src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -197,11 +201,16 @@ internal partial class CodeFixService : ICodeFixService
 
             // append fixes for all diagnostics with the same diagnostics span
             using var resultDisposer = ArrayBuilder<CodeFixCollection>.GetInstance(out var result);
-            foreach (var spanAndDiagnostic in aggregatedDiagnostics)
+
+            // 'CodeActionRequestPriority.Low' is used when the client only wants suppression/configuration fixes.
```

Comment:
i feel like this is dangerous.  we're encoding knowledge about a high level concept at a lower level.  I would prefer that hte lower levers just somehow specify their own buckets (like 'low') and then things naturally fall into that.

Reply:
All the reasoning about semantics of CodeActionRequestPriority is now encoded in this method alone. Higher layers just pass in CodeActionRequestPriority down to this method, which determines what fixes/analyzers to execute/skip for the given priority.